### ISSUE TITLE
Conflicting peer dependency: @lingui/babel-plugin-lingui-macro@5.1.0

### ISSUE THREAD

**Issue by Athou** [lingui/js-lingui#2124]
**Describe the bug**
Cannot upgrade lingui to 5.1.1 from 5.0.0.

```
[INFO] npm error While resolving: @lingui/core@5.1.1
[INFO] npm error Found: @lingui/babel-plugin-lingui-macro@5.1.1
[INFO] npm error node_modules/@lingui/babel-plugin-lingui-macro
[INFO] npm error   dev @lingui/babel-plugin-lingui-macro@"^5.1.1" from the root project
[INFO] npm error   @lingui/babel-plugin-lingui-macro@"5.1.1" from @lingui/cli@5.1.1
[INFO] npm error   node_modules/@lingui/cli
[INFO] npm error     dev @lingui/cli@"^5.1.1" from the root project
[INFO] npm error     @lingui/cli@"5.1.1" from @lingui/vite-plugin@5.1.1
[INFO] npm error     node_modules/@lingui/vite-plugin
[INFO] npm error       dev @lingui/vite-plugin@"^5.1.1" from the root project
[INFO] npm error
[INFO] npm error Could not resolve dependency:
[INFO] npm error peerOptional @lingui/babel-plugin-lingui-macro@"5.1.0" from @lingui/core@5.1.1
[INFO] npm error node_modules/@lingui/core
[INFO] npm error   @lingui/core@"^5.1.1" from the root project
[INFO] npm error   @lingui/core@"5.1.1" from @lingui/babel-plugin-lingui-macro@5.1.1
[INFO] npm error   node_modules/@lingui/babel-plugin-lingui-macro
[INFO] npm error     dev @lingui/babel-plugin-lingui-macro@"^5.1.1" from the root project
[INFO] npm error     1 more (@lingui/cli)
[INFO] npm error   2 more (@lingui/cli, @lingui/react)
[INFO] npm error
[INFO] npm error Conflicting peer dependency: @lingui/babel-plugin-lingui-macro@5.1.0
[INFO] npm error node_modules/@lingui/babel-plugin-lingui-macro
[INFO] npm error   peerOptional @lingui/babel-plugin-lingui-macro@"5.1.0" from @lingui/core@5.1.1
[INFO] npm error   node_modules/@lingui/core
[INFO] npm error     @lingui/core@"^5.1.1" from the root project
[INFO] npm error     @lingui/core@"5.1.1" from @lingui/babel-plugin-lingui-macro@5.1.1
[INFO] npm error     node_modules/@lingui/babel-plugin-lingui-macro
[INFO] npm error       dev @lingui/babel-plugin-lingui-macro@"^5.1.1" from the root project
[INFO] npm error       1 more (@lingui/cli)
[INFO] npm error     2 more (@lingui/cli, @lingui/react)
[INFO] npm error
[INFO] npm error Fix the upstream dependency conflict, or retry
[INFO] npm error this command with --force or --legacy-peer-deps
[INFO] npm error to accept an incorrect (and potentially broken) dependency resolution.
```

**To Reproduce**
Bump all those lingui packages to 5.1.1

![image](https://github.com/user-attachments/assets/9f9061a4-be34-4d7b-be96-4508780d7c1d)

See also this renovate PR: https://github.com/Athou/commafeed/pull/1623/files

**Expected behavior**
`npm install` works without error.

**Additional context**
Could the issue be that the babel-plugin-lingui-macro version [here](https://github.com/lingui/js-lingui/blob/main/packages/core/package.json#L72) is pinned to 5.1.0?

**Comment by andrii-bodnar** [comment]
It appears that this is related to Lerna, particularly this issue - https://github.com/lerna/lerna/issues/955 😞 

It doesn't bump `peerDependencies` during the publishing process. I will look into this.

**Comment by timofei-iatsenko** [comment]
@andrii-bodnar I propose to change it to the [changesets](https://github.com/changesets/changesets).

**Comment by andrii-bodnar** [comment]
@timofei-iatsenko I also think it's time to move to changesets.

**Comment by jgarplind** [comment]
6 hours late to the party, I agree with your conclusion, but understand that it may be side work that you don't really want to spend time on.

So I just wanted to chime in and let you know that there are more of us (probably a huge silent majority too) who would be even happier with Lingui if you did get this figured out.

**Comment by andrii-bodnar** [comment]
I just released a new version with the fix - [5.1.2](https://github.com/lingui/js-lingui/releases/tag/v5.1.2). Sorry for the inconvenience!
